Satellite-im/UplinkWeb
# chore(profile): delete account

### What this PR does 📖

https://github.com/user-attachments/assets/9767b9ff-84a0-4656-aa06-d405981c8f09


User can delete account on profile.

How to test
- sign up
- profile
- delete account
- account should delete properly and go again automatically to create new account and user should be able to do a new account as well
